Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/Controls/Circle.cs

```diff
@@ -1,17 +1,13 @@
 ﻿namespace Meadow.Foundation.Graphics.MicroLayout;
 
-public struct Coordinate2D
-{
-    public int X { get; set; }
-    public int Y { get; set; }
-}
-
 /// <summary>
 /// Represents a circle in the user interface.
 /// </summary>
 public class Circle : ThemedControl
 {
     private Color _foreColor;
```

Comment:
you heard the boss - underscores need to go

Reply:
Doh!  Muscle memory